README.md

```diff
@@ -1,3 +1,8 @@
+> [!Important]
+> Linebender no longer uses Glazier, and so this project is no longer supported.
+>
+> You should prefer to use [winit](https://crates.io/crates/winit) instead.
```

Comment:
Should `winit` be in backticks here like it is below?

Reply:
I could make a claim that one is talking about the crate whilst the other is talking about the project. But I've just removed the backticks